great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -57,8 +57,10 @@ def __init__(
             Dict[str, Union[ExpectationSuiteValidationResult, dict, str]],
         ],
         checkpoint_config: CheckpointConfig,
+        validation_result_url: Optional[str] = None,
         success: Optional[bool] = None,
     ) -> None:
+        self._validation_result_url = validation_result_url
```

Comment:
Do we need to make a corresponding change to `CheckpointResultSchema`? See `checkpoint/types/checkpoint_result.py`

Reply:
I don't think this schema is being used for some reason at least not in this file. We tried to implement this change with @NathanFarmer but it was giving us an error. I decided to leave this out for now.